<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PhyxCalc</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/appicon/icon</normaloff>:/appicon/icon</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QHidingTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>768</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <widget class="QMenu" name="menuDocks">
     <property name="title">
      <string>&amp;Docks</string>
     </property>
     <addaction name="actionVariables"/>
     <addaction name="actionConstants"/>
     <addaction name="actionUnits"/>
     <addaction name="actionPrefixes"/>
     <addaction name="actionFunctions"/>
    </widget>
    <addaction name="actionNew_Tab"/>
    <addaction name="menuDocks"/>
    <addaction name="action_Slim_Mode"/>
    <addaction name="action_Plot_Window"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuOpen_Recent">
     <property name="title">
      <string>Open &amp;Recent</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/icons/images/document-open-recent.png</normaloff>:/icons/images/document-open-recent.png</iconset>
     </property>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="menuOpen_Recent"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionSave_All"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionClose_Other"/>
    <addaction name="actionClose_All"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuCalculation">
    <property name="title">
     <string>&amp;Calculation</string>
    </property>
    <addaction name="actionRecalculate_All"/>
    <addaction name="actionRecalculate_from_Line"/>
    <addaction name="actionClear_Variables"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuCalculation"/>
   <addaction name="menuWindow"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="variablesDock">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Variables</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="margin">
      <number>0</number>
     </property>
     <item row="0" column="0" colspan="2">
      <widget class="QTableWidget" name="variableTable">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="constantsDock">
   <property name="windowTitle">
    <string>Constants</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QTableWidget" name="constantsTable">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="unitsDock">
   <property name="windowTitle">
    <string>Units</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTableWidget" name="unitsTable"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="functionsDock">
   <property name="windowTitle">
    <string>Functions</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QListWidget" name="functionsList"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="prefixesDock">
   <property name="windowTitle">
    <string>Prefixes</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTableWidget" name="prefixesTable"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/document-new.png</normaloff>:/icons/images/document-new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/document-open.png</normaloff>:/icons/images/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/document-save.png</normaloff>:/icons/images/document-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/document-save-as.png</normaloff>:/icons/images/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As..</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/document-save-all.png</normaloff>:/icons/images/document-save-all.png</iconset>
   </property>
   <property name="text">
    <string>Save A&amp;ll</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/dialog-close.png</normaloff>:/icons/images/dialog-close.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionClose_All">
   <property name="text">
    <string>Clos&amp;e All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/application-exit.png</normaloff>:/icons/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/edit-undo.png</normaloff>:/icons/images/edit-undo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/edit-redo.png</normaloff>:/icons/images/edit-redo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionVariables">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Variables</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/document-export.png</normaloff>:/icons/images/document-export.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Export..</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/configure.png</normaloff>:/icons/images/configure.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About PhyxCalc..</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/edit-cut.png</normaloff>:/icons/images/edit-cut.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/edit-copy.png</normaloff>:/icons/images/edit-copy.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;opy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/edit-paste.png</normaloff>:/icons/images/edit-paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionClose_Other">
   <property name="text">
    <string>Close Other</string>
   </property>
  </action>
  <action name="actionRecalculate_All">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/run-build.png</normaloff>:/icons/images/run-build.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Recalculate All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionRecalculate_from_Line">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/run-build-cursor.png</normaloff>:/icons/images/run-build-cursor.png</iconset>
   </property>
   <property name="text">
    <string>Recalculate from &amp;Line</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionConfigure_and_control">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/configure.png</normaloff>:/icons/images/configure.png</iconset>
   </property>
   <property name="text">
    <string>Configure and control PhyxCalc</string>
   </property>
   <property name="toolTip">
    <string>Configure and control PhyxCalc</string>
   </property>
  </action>
  <action name="actionPhyxCalc">
   <property name="text">
    <string>PhyxCalc</string>
   </property>
  </action>
  <action name="action_Slim_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Slim Mode</string>
   </property>
  </action>
  <action name="actionConstants">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Constants</string>
   </property>
  </action>
  <action name="actionClear_Variables">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/edit-clear.png</normaloff>:/icons/images/edit-clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear Variables</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionUnits">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Units</string>
   </property>
  </action>
  <action name="actionFunctions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Functions</string>
   </property>
  </action>
  <action name="actionPrefixes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Prefixes</string>
   </property>
  </action>
  <action name="actionNew_Tab">
   <property name="text">
    <string>&amp;New Tab</string>
   </property>
  </action>
  <action name="action_Plot_Window">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Plot Window</string>
   </property>
  </action>
  <zorder>prefixesDock</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QHidingTabWidget</class>
   <extends>QTabWidget</extends>
   <header>qhidingtabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
